javascript-arrays/javascript-arrays-quiz-notes.md

```diff
@@ -0,0 +1,45 @@
+# javascript-arrays-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What are arrays used for?
+
+- Describe array literal notation.
+
+- How are arrays different from "plain" objects?
+
+- What number represents the first index of an array?
+
+- What is the `length` property of an array?
+
+- How do you calculate the last index of an array?
```

Comment:
Don't forget to answer the quiz questions!

Reply:
sorry, forgot to push them!